src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php

```diff
@@ -634,10 +634,10 @@ public function validateSchema(\DOMDocument $dom)
 EOF
         ;
 
-        if (\LIBXML_VERSION < 20900) {
-            $disableEntities = libxml_disable_entity_loader(false);
+        if (\PHP_VERSION_ID < 90000) {
```

Comment:
We don't know anything about a potential php 9. I'd either remove the check completely or use `function_exists()` instead.